charts/db-instances/templates/_helpers.tpl

```diff
@@ -3,7 +3,7 @@
 Expand the name of the chart.
 */}}
 {{- define "db-instances.name" -}}
```

Comment:
actually "db-instances.name" doesn't make sense since it's using value `serviceMonitorName`.
Maybe we can use "db-instances.smName" something?